.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Feature Status
```

Comment:
We typically indicate that a PR is a "work in progress" by putting "[WIP]" in the title of the PR. We remove that once it's ready to merge (in theory).

Also, not all PRs are to add a new feature. Some are to fix a bug. Some are to add/update documentation. Some are to bump a version number. And so on.